configs/mmdet/mmdet_regression_test.yaml

```diff
@@ -0,0 +1,175 @@
+globals:
+  codebase_name: mmdet
+  codebase_release: V2.22.0
```

Comment:
maybe we should remove version info and assume it is determined by user's env.

Reply:
Done